docs/src/howto/external-datastore.md

```diff
@@ -0,0 +1,80 @@
+# How to use an external datastore
+
+Canonical Kubernetes supports using an external datastore such as etcd
+instead of the bundled dqlite datastore.
+This guide walks you through configuring an external etcd datastore.
+
+## What you'll need
+
+This guide assumes the following:
+
+- You have root or sudo access to the machine
+- You have an external etcd cluster
+- You have installed the Canonical Kubernetes snap
+  (see How-to [Install Canonical Kubernetes from a snap][snap-install-howto]).
+- You have not bootstrapped the Canonical Kubernetes cluster yet
+
+```{warning}
+The selection of the backing datastore can only be changed during the bootstrap process.
+There is no migration path between the bundled dqlite and the external datastores.
+```
+
+## Adjust the bootstrap configuration
+
+To use an external datastore, a configuration file that contains the required
+datastore parameters needs to be provided to the bootstrap command.
+Create a configuration file and insert the contents below while replacing
```

Comment:
Do we have a reference to a complete bootstrap config file somewhere in our docs?

Reply:
I don't think we have a Reference page for the bootstrap config file currently, I'll create a card if there isn't one already.